bookkeeper-server/src/main/java/org/apache/bookkeeper/bookie/Journal.java

```diff
@@ -196,11 +196,11 @@ void rollLog(LastLogMark lastMark) throws NoWritableLedgerDirException {
             // which is safe since records before lastMark have been
             // persisted to disk (both index & entry logger)
             lastMark.getCurMark().writeLogMark(bb);
-            if (LOG.isDebugEnabled()) {
-                LOG.debug("RollLog to persist last marked log : {}", lastMark.getCurMark());
-            }
+
+            LOG.info("RollLog to persist last marked log : {}", lastMark.getCurMark());
```

Comment:
@reddycharan 
I switched this log line from `DEBUG` to `INFO` since I believe that this is an important event to be logged. However given that we use ELPL, which flushes every 10 seconds, is this a problem?

Reply:
yes, having log line for every 10 secs is not ideal. Keep it at debug level.